Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -103,11 +110,13 @@ class TextureCacheBase
 
     ~TCacheEntry();
 
-    void SetGeneralParameters(u32 _addr, u32 _size, TextureAndTLUTFormat _format)
+    void SetGeneralParameters(u32 _addr, u32 _size, TextureAndTLUTFormat _format,
```

Comment:
the underscore should be after the name, not before it.

Reply:
This was just how it was, so I left it.  There are other functions in this file that use the same notation so it probably makes sense to clean them up with a separate PR.